mne/io/snirf/_snirf.py

```diff
@@ -0,0 +1,202 @@
+# Authors: Robert Luke <mail@robertluke.net>
+#
+# License: BSD (3-clause)
+
+import re as re
+import numpy as np
+
+from ..base import BaseRaw
+from ..meas_info import create_info
+from ...annotations import Annotations
+from ...utils import logger, verbose, fill_doc, warn, requires_h5py
+
+
+@requires_h5py
+@fill_doc
+def read_raw_snirf(fname, preload=False, verbose=None):
+    """Reader for a SNIRF fNIRS recording.
```

Comment:
Maybe for now:
```suggestion
    """Reader for a continuous wave SNIRF data.
```

Then the `.. note::` below can live in a standalone `Notes` section?